example/src/main/assets/mobile-services.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": 1,
+    "clusterName": "https://192.168.37.1:8443",
+    "namespace": "myproject",
+    "clientId": "app-cordova",
+    "services": [
+        {
+            "id": "keycloak",
+            "name": "keycloak",
+            "type": "keycloak",
+            "url": "https://keycloak.security.feedhenry.org",
+            "config": {
+                "auth-server-url": "https://keycloak.security.feedhenry.org/auth",
+                "clientId": "client-app",
```

Comment:
This is duplicated with the resource field below. I think we can remove this one as Keycloak uses `resource` in its Keycloak JSON installation format. We will probably need to double check if other code is referring to this and make it refer to `resource` instead of `clientId`.

Reply:
will remove it in a separate PR.